### ISSUE TITLE
[WIP] French translation

### ISSUE THREAD

**Issue by DarthJahus** [kvhnuke/etherwallet#1193]
Many of the French strings have to be changed. I started reviewing them and correcting/changing the sentences one by one.

There are 674 lines in the file, so the process will need a lot of time. Will subit a Pull Request once done.

I felt like I had to open this discussion in case there are information I have to know before going further.

**Comment by DarthJahus** [comment]
I need details about this string:

`ADD_Radio_5_Path            : 'Select HD derivation path',`

I have no idea what "HD derivation path" can mean in this context.

**Comment by MysticRyuujin** [comment]
Ok I might not be 100% here on the wording/logic but if you have a hardware wallet it can store different private keys for different coins in different locations on the device and it's asking you to select the path you want to connect to. I have a Ledger and when I try to connect to it I'm presented with the following screenshot:

![image](https://user-images.githubusercontent.com/8877131/32742785-01debfd4-c870-11e7-98ef-afad467ed33d.png)

So basically it's asking you which path on the device you want to open, because one device can contain many different keys for different coins.

Does that help @DarthJahus ?

**Comment by DarthJahus** [comment]
It does! Thank you very much, @MysticRyuujin.

**Comment by DarthJahus** [comment]
I got a question about `PARITY_GasLimitExceeded`. 

It says: `"Transaction cost exceeds current gas limit. Limit: {}, got: {}. Try decreasing supplied gas.",`

If transaction cost exceeds current gas limit, shouldn't we rise the supplied gas?

**Comment by MysticRyuujin** [comment]
Yes, but I it's up to the user to specify a max gas (note this is not gas
price but gas LIMIT) the gas limit is a feature to protect against loops
and runaway code.

**Comment by DarthJahus** [comment]
Understood that. But it says "try decreasing supplied gas". Shouldn't it be "try increasing supplied gas"?

**Comment by MysticRyuujin** [comment]
I wonder if it's talking about the current  block gas limit? I bet that's
what it is...which I THINK is 7 million right now.
